clouds/databricks/Makefile

```diff
@@ -88,3 +97,13 @@ ci-release-core:
 
 create-headers:
 	$(MAKE) -C libraries/scala create-headers
+
+create-package: clean build
```

Comment:
Needs to be a production build. Why clean is added here?

Suggested

```
create-package:
        $(MAKE) build production=1
        ...
```

Reply:
🤔 I thought I had put the production in the CI call (since this is intended to be used from CI), but yes, better add it here. The clean was probably from avoiding old files in local tests, removing it

Reply:
This is what we are doing in other clouds: no clean and build with production=1 to avoid generating packages with custom prefix